pp.go

```diff
@@ -129,8 +131,20 @@ func ResetColorScheme() {
 	currentScheme = defaultScheme
 }
 
+// WithLineInfo add file name and line information to output
+// call this function with care, because getting stack has performance penalty
+func WithLineInfo(w bool) {
```

Comment:
Why didn't you let `withLineInfo` just `WithLineInfo`, like `ColoringEnabled`? In my opinion,

1. Interface should be unified.
2. This configuration is simple, and I think there will be no need to abstract this configuration as function and make this variable private.